<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spoton_encryptfile</class>
 <widget class="QMainWindow" name="spoton_encryptfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icons/icons.qrc">
    <normaloff>:/Logo/goldbug.png</normaloff>:/Logo/goldbug.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="banderole_chat">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.023, y1:0.988636, x2:0.028, y2:0.011, stop:0 rgba(218, 232, 255, 255), stop:1 rgba(170, 85, 255, 255));

color: rgb(255, 255, 255);


</string>
      </property>
      <property name="text">
       <string>File Encryption</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>&amp;Destination</string>
        </property>
        <property name="buddy">
         <cstring>destination</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="destination"/>
      </item>
      <item>
       <widget class="QPushButton" name="selectDestination">
        <property name="styleSheet">
         <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; } 


QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 }
</string>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Origin</string>
        </property>
        <property name="buddy">
         <cstring>file</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="file"/>
      </item>
      <item>
       <widget class="QPushButton" name="select">
        <property name="styleSheet">
         <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; } 


QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 }
</string>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Secret</string>
        </property>
        <property name="buddy">
         <cstring>password</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Minimum of 16 characters.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>P&amp;IN</string>
        </property>
        <property name="buddy">
         <cstring>pin</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pin"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QRadioButton" name="decrypt">
        <property name="text">
         <string>&amp;Decrypt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="encrypt">
        <property name="text">
         <string>&amp;Encrypt</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&amp;Cipher Type</string>
        </property>
        <property name="buddy">
         <cstring>cipher</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cipher">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
         <item>
          <widget class="QRadioButton" name="cbc">
           <property name="text">
            <string>&amp;CBC</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ctr">
           <property name="text">
            <string>&amp;CTR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="gcm">
           <property name="text">
            <string>&amp;GCM</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&amp;Hash Type</string>
        </property>
        <property name="buddy">
         <cstring>hash</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="hash">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="styleSheet">
         <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; } 


QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 }
</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sign">
        <property name="text">
         <string>&amp;Sign file.</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="convert">
        <property name="styleSheet">
         <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; } 


QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 }
</string>
        </property>
        <property name="text">
         <string>Convert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset">
        <property name="styleSheet">
         <string notr="true">QPushButton {
 border: 2px solid gray;
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border-radius: 5px;
 margin: 2px;
 padding: 2px;
 }


QPushButton:hover { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 rgb(254,229,172), stop: 1 #FFFFFF);
background-position: top center; } 


QPushButton:pressed {
     background-color: darkgrey;
     border-style: inset;
 }
</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Close"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Empty_Log">
   <property name="text">
    <string>&amp;Empty Log</string>
   </property>
  </action>
  <action name="actionEnable_Log">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>E&amp;nable Log</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>destination</tabstop>
  <tabstop>selectDestination</tabstop>
  <tabstop>file</tabstop>
  <tabstop>select</tabstop>
  <tabstop>password</tabstop>
  <tabstop>pin</tabstop>
  <tabstop>decrypt</tabstop>
  <tabstop>encrypt</tabstop>
  <tabstop>cipher</tabstop>
  <tabstop>cbc</tabstop>
  <tabstop>ctr</tabstop>
  <tabstop>gcm</tabstop>
  <tabstop>hash</tabstop>
  <tabstop>cancel</tabstop>
  <tabstop>sign</tabstop>
  <tabstop>convert</tabstop>
  <tabstop>reset</tabstop>
 </tabstops>
 <resources>
  <include location="../Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
